### DISCUSSION TITLE
FCRT

### DISCUSSION THREAD

**Discussion by Klesone** [hermonochy/Rescueland#31] [Q&A] [answered]
How are FCRTs made?

**Comment by hermonochy** [comment, marked as answer]
FCRTs are created in a factory similar to other robots, but they initially lack programming. If designated for military service, [DarkArmour](./The%20Technology%20of%20Rescueland%2C%20Computerland%20and%20it's%20surrounding%20Planets.md#fcrt-darkarmour-r) is then installed. Following that, Rescueland Gas is injected into their main computer system. An FCRT is made!
